python/ray/serve/api.py

```diff
@@ -711,15 +711,18 @@ def start(
 
     register_custom_serializers()
 
+    try:
+        _get_global_client()
+        logger.info("Connecting to existing Serve instance.")
+        return
+    except RayServeException:
+        pass
+
     # Try to get serve controller if it exists
     if detached:
         controller_name = SERVE_CONTROLLER_NAME
         try:
             ray.get_actor(controller_name)
-            raise RayServeException("Called serve.start(detached=True) but a "
-                                    "detached instance is already running. "
-                                    "Please use serve.connect() to connect to "
-                                    "the running instance instead.")
         except ValueError:
             pass
```

Comment:
It looks like this try-except block isn't doing anything now, should we get rid of it entirely?

Reply:
Ah yes we should...